includes/admin/class-wc-admin-duplicate-product.php

```diff
@@ -123,6 +123,7 @@ public function product_duplicate( $product ) {
 		}
 		$duplicate->set_status( 'draft' );
 		$duplicate->set_date_created( null );
+		$duplicate->set_slug( null );
```

Comment:
Yea you're right, the docs say the expected type is a string. I've changed it now.
Although I think null is more self-explanatory for such cases. (empty strings looks to me like a mistake at first glance)